src/marqo/core/models/marqo_index_request.py

```diff
@@ -46,13 +47,21 @@ class FieldRequest(StrictBaseModel):
     type: marqo_index.FieldType
     features: List[marqo_index.FieldFeature] = []
     dependent_fields: Optional[Dict[str, float]] = pydantic.Field(alias='dependentFields')
-
+    
+    @validator('type', pre=True, always=True)
+    def normalize_type(cls, v):
+        # Normalize the type string by removing extra spaces around '<', '>', and ', '
+        normalized_type = re.sub(r'\s*<\s*', '<', v)
+        normalized_type = re.sub(r'\s*>\s*', '>', normalized_type)
+        normalized_type = re.sub(r'\s*,\s*', ', ', normalized_type)
+        return marqo_index.FieldType(normalized_type)
```

Comment:
I am not sure if it is the best practice here to change the type of value from string to enum. Maybe just return `normalized_type`?